datafusion/core/src/execution/context.rs

```diff
@@ -1153,7 +1103,7 @@ pub struct SessionConfig {
     /// due to `resolve_table_ref` which passes back references)
     default_schema: String,
     /// Configuration options
-    config_options: ConfigOptions,
+    options: ConfigOptions,
```

Comment:
👎  for the change to the more overloaded `options` name, even though I realize it is more concise

Reply:
Yeah, fustfmt was being obtuse and formatting things over 7 lines and this made it happy. SessionConfig isn't long for this world so I think it is fine